dev/package-examples/auditd-2.0.4/docs/README.md

```diff
@@ -34,51 +33,45 @@ registered to receive data from the Linux Audit Framework. You can use these
 commands to see if the `auditd` service is running and stop it:
 
 * See if `auditd` is running:
-+
-[source,shell]
------
+
+```
 service auditd status
------
+```
 
 * Stop the `auditd` service:
-+
-[source,shell]
------
+
+```
 service auditd stop
------
+```
 
 * Disable `auditd` from starting on boot:
-+
-[source,shell]
------
+
+```
 chkconfig auditd off
------
+```
 
 To save CPU usage and disk space, you can use this command to stop `journald`
 from listening to audit messages:
 
-[source,shell]
------
+```
 systemctl mask systemd-journald-audit.socket
------
+```
+
 
-[float]
-=== Inspect the kernel audit system status
+## Inspect the kernel audit system status
 
-{beatname_uc} provides useful commands to query the state of the audit system
+The agent provides useful commands to query the state of the audit system
 in the Linux kernel.
 
 * See the list of installed audit rules:
-+
-[source,shell]
------
+
+```
 auditbeat show auditd-rules
```

Comment:
@ph Will there be  a way with the agent to still call these commands indirectly?